validator/client/src/main/java/tech/pegasys/teku/validator/client/KeyManager.java

```diff
@@ -13,12 +13,73 @@
 
 package tech.pegasys.teku.validator.client;
 
+import java.util.List;
 import java.util.Set;
+import org.apache.commons.lang3.NotImplementedException;
 import tech.pegasys.teku.bls.BLSPublicKey;
 import tech.pegasys.teku.validator.client.loader.ValidatorLoader;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.ActiveValidator;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.DeleteKeysResponse;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.ImportStatus;
 
 public class KeyManager {
 
+  /**
+   * Get a listing of active validator keys
+   *
+   * @return a list of active validators
+   */
+  List<ActiveValidator> getActiveValidatorKeys() {
+    throw new NotImplementedException("getActiveValidatorKeys not implemented yet");
+  }
+
+  /**
+   * Delete a collection of validators
+   *
+   * <p>The response must be symmetric, the list of validators coming in dictates the response
+   * order.
+   *
+   * <p>An individual deletion failure MUST NOT cancel the operation, but rather cause an error for
+   * that specific key Validator keys that are reported as deletionStatus.deleted MUST NOT be active
+   * once the result is returned.
+   *
+   * <p>Each failure should result in a failure message for that specific key Slashing protection
+   * data MUST be returned if we have the information
+   *
+   * <p>- NOT_FOUND should only be returned if the key was not there, and we didn't have slashing
+   * protection information
+   *
+   * <p>- NOT_ACTIVE indicates the key wasn't active, but we had slashing data
+   *
+   * <p>- DELETED indicates the key was found, and we have stopped using it and removed it.
+   *
+   * <p>- ERROR indicates we couldn't stop using the key (read-only might be a reason)
+   *
+   * <p>If an exception is thrown, it will cause a 500 error on the API, so that is an undesirable
+   * outcome.
+   *
+   * @param validators list of validator public keys that should be removed
+   * @return The result of each deletion, and slashing protection data
+   */
+  public DeleteKeysResponse deleteValidators(final List<BLSPublicKey> validators) {
+    throw new NotImplementedException("deleteValidators not implemented yet");
+  }
+
+  /**
+   * import a collection of validators The result needs to be symmetric with the input order Must *
+   * supply a message in case of error. Super important that an individual error doesn't fail the *
+   * whole process, each should be its own process that can error out.
```

Comment:
Not a big deal but the formatting on this one didn't work out (and should start with a capital I).